code/modules/mob/living/carbon/human/human_emote.dm

```diff
@@ -31,6 +31,34 @@
 			return TRUE
 	return ..()
 
+/datum/emote/living/carbon/human/hiss
+	key = "hiss"
+	key_third_person = "hisses"
+	message = "hisses."
+	message_mime = "hisses silently."
+	message_param = "hisses at %t."
+	emote_type = EMOTE_AUDIBLE | EMOTE_MOUTH
+	muzzled_noises = list("weak hissing", "irritated")
+
+/datum/emote/living/carbon/human/hiss/get_sound(mob/living/user)
+	var/mob/living/carbon/human/H = user
+	if(!istype(H) || H.mind?.miming)
+		return
+	return H.dna.species.hiss_sound
+
+/datum/emote/living/carbon/human/hiss/burn_hiss
+	key = "burn_hiss"
+	key_third_person = "burn_hisses"
+	message = "hisses as their skin peels off!"
+	message_mime = "grimaces as their skin peels off!."
+	emote_type = EMOTE_AUDIBLE | EMOTE_MOUTH
+	muzzled_noises = list("pained hissing")
+
+/datum/emote/living/carbon/human/burn_hiss/run_emote(mob/user, params, type_override, intentional)
+	var/mob/living/carbon/human/H = user
+	if(!istype(H) || !intentional)
+		return
```

Comment:
Is this doing anything? I think you might need to call parent here

Reply:
During testing my skrell hisses normally, so I don't know?
I can just call parent here as well

Reply:
I think what you *want* is to check the condition before making the parent call. If the condition is false, just return FALSE, and after that conditional `return ..()`. That make sense?